### ISSUE TITLE
error: unknown option '--mode=build'

### ISSUE THREAD

**Issue by me-vlad-k** [nwutils/nw-builder#1277]
### Issue Type

- [x] Bug Report

### Current/Missing Behaviour
I can't build the application, regardless of which CLI options I choose. It shows the error `error: unknown option '--mode=build'`. This error appears for any option that comes first. It occurs in all versions from 4.10.0 up to 4.11.4, which is the current version.

### Expected/Proposed Behaviour
The application should build normally, which I can successfully do in versions 4.8.1 and 4.9.0. I haven't tested lower versions.

### Additional Info

- Package version: 4.10.0 to 4.11.0
- Operating System: macOS Sonoma 14.6.1 
- Comupter: MacBook Pro M1 
- Node version: 22.7.0
- NW.js version: 0.92.0
- Repo: https://github.com/me-vlad-k/nwbuilder-issue-example.git

I am using a very basic demo that works well in versions 4.8.1 and 4.9.0.

**Comment by ayushmanchhabra** [comment]
In the process of fixing this in #1255

**Comment by ayushmanchhabra** [comment]
In the meantime, you could use the JavaScript API

**Comment by me-vlad-k** [comment]
@ayushmanchhabra  Thanks for looking into this issue. I'm really glad that nw-builder supports the ARM64 platform for macOS. I decided to switch from [nwjs-builder-phoenix](https://github.com/evshiron/nwjs-builder-phoenix) since it's pretty outdated now. I was about to pull my hair out trying to get nw-builder to work 😆 Thanks to Jared Wilcurt's [nw-angular-example](https://github.com/nwutils/nw-angular-example), I realized it was a version issue, not the settings I was using, since his example was running 4.8.1 without any problems. I really appreciate the suggestion about the JavaScript API! For now, I've downgraded to version 4.9.0, and it’s working fine for me. I'm sure the update will take care of the problem once it's released. I'll keep an eye out for it!
